### ISSUE TITLE
My configs (not ip banned ATM)

### ISSUE THREAD

**Issue by RoseCorpse** [finndev/PokeBuddy#894]
As I said in a post which number I cant remember. my bot is working 90% fine (just a little slow at pokemon catching) and I'm gonna share with you my configs, let me know if it worked for you too.

- Catch Pokemon
- Farm Pokestops
- Recycling items
- Evolving pokemon (no lucky eggs)
- Use incubator (perma one)
- Transfer bad pokemon.
- 15km/h (40k exp/h)

I'm using Nox, not cellphone, same coordinates in both bot and nox, have 2 pokemons at gyms in my country and teleport to Venezuela once a day just to check gyms/own gyms. If I want to check my char, I see my pokebuddy last coordinate and I copy/paste it in nox so I can appear at the same spot that was pokebuddy before I stop it.

Let me know if it worked for you too.

**Comment by shiitan5** [comment]
whats nox ?!

**Comment by RoseCorpse** [comment]
http://en.bignox.com/#p1

**Comment by RoseCorpse** [comment]
Android Emulator

**Comment by 2stronkkk** [comment]
ALTITUDE?

**Comment by RoseCorpse** [comment]
@2stronkkk leaved it in 10

**Comment by 2stronkkk** [comment]
ty. i am gonna test. i am always ban after 5 minutes playing

**Comment by 2stronkkk** [comment]
banned after 5 minutes again :(

**Comment by RoseCorpse** [comment]
damn!! I dont know then u.u

**Comment by RoseCorpse** [comment]
[img]https://lh3.googleusercontent.com/-gQUnuz02xwk/Va7can5SMvI/AAAAAAAABcA/dP0CTVNAXs0/w500-h281/a4d44151394ebabea818c613d39f75f1.gif[/img]

**Comment by RoseCorpse** [comment]
[img]https://lh3.googleusercontent.com/-gQUnuz02xwk/Va7can5SMvI/AAAAAAAABcA/dP0CTVNAXs0/w500-h281/a4d44151394ebabea818c613d39f75f1.gif[/img]

**Comment by PedroPovoleri** [comment]
Strange, in brazil is ok, i was play teleporting and make other stuffs on here util now i was walkend for 8 hours [img]blob:https://drive.google.com/11a047ae-764e-4367-b2fe-911fba98c4e5[img]

**Comment by PedroPovoleri** [comment]
![capturar](https://cloud.githubusercontent.com/assets/2331743/17607788/5ff3fb56-5ffe-11e6-82ee-265c0e251bf2.PNG)

Here img

**Comment by stavrosian** [comment]
same 2stronk, i can play for maybe 10 minutes tops.

**Comment by tsebis** [comment]
40/xp per hour? so low, i can get more with fake gps xd

**Comment by RoseCorpse** [comment]
it's a little slow catching

**Comment by horseyMEAT** [comment]
Yup, I get banned after 10 or so minutes of being extremely careful. 
Can't figure it out, I don't walk too fast, I don't teleport, it's something, we gotta figure this out.

**Comment by ChiisanaKoneko** [comment]
@horseyMEAT maybe its your location do you ever load it on a phone or something after using the bot if so if the location is different then youll get banned

**Comment by horseyMEAT** [comment]
@ChiisanaKoneko nope, I am very careful and make sure I do everything "perfect". 
And I still get banned.

**Comment by ChiisanaKoneko** [comment]
@horseyMEAT i haven't got banned once and ive been using it for days, so if your saying that its very confusing lol

**Comment by horseyMEAT** [comment]
@ChiisanaKoneko dang, I wish. Where you to GPS to?
I am so careful, I cannot figure out how it's "catching" me. 
I always wait long enough before checking in on a different device. 
I GPS to Central Park and I live 1.5 hrs from it. I make sure to never make it look like I teleported. 
Gotta be able to figure this out.

**Comment by ChiisanaKoneko** [comment]
@horseyMEAT i don't gps to random locations that's the problem you probably have i set default location to my house so it runs from my home and when im done i can turn on my phone to evolve etc

**Comment by horseyMEAT** [comment]
@ChiisanaKoneko 
I can't do it from home, the closest items are about 7 miles away. 
Also I have done in it my local town (which sucks) and it still gets banned after 10-15 min of use. 
Just now tried north part of Central Park, worked great for about 10-12 min, now banned again.

**Comment by ChiisanaKoneko** [comment]
@horseyMEAT  its a few miles away for items around my house the bot works and gets stuck but after a few mins walks to other parts so id advise you to try and run from your house location that way you can use your phone without and gps spoof so it looks normal i don't know how you can get banned so much when i haven't got banned once on this game and i play every few mins

**Comment by horseyMEAT** [comment]
@ChiisanaKoneko tried it. Didn't work. It just sat still and never moved. :(

**Comment by horseyMEAT** [comment]
Waited till ban was lifted again. This time it lasted 6 minutes till banned.

**Comment by ChiisanaKoneko** [comment]
@horseyMEAT im sorry i dont know what to do then mine are just in the radius it moves i guess and i dont know what could be done to make it work for you

**Comment by horseyMEAT** [comment]
Thanks anyway though :)
